Examples/src/components/Examples/Charts2D/BasicChartTypes/RealTimeMountainChart/angularrealTimeMountainapp.ts

```diff
@@ -0,0 +1,42 @@
+import { Component } from "@angular/core";
+import { SciChartSurface, SciChart3DSurface } from "scichart";
+
+// @ts-ignore
+import { drawExample } from "./drawExample";
+
+SciChartSurface.loadWasmFromCDN();
+SciChart3DSurface.loadWasmFromCDN();
+
+@Component({
+    selector: "app-realTimeMountain-chart",
+    template: `<scichart-angular
+    [initChart]="drawExample"
+    (onInit)="onInitHandler($event)"
+    (onDelete)="onDeleteHandler($event)"
+></scichart-angular>`,
+})
+export class AppRealTimeMountainComponent {
+    title = "scichart-angular-app";
+
+    private sciChartSurface:any;
+    
+    drawExample = drawExample;
+    
+    onInitHandler = (initResult: any) => {
+        initResult.controls.handleStart();
+    };
+  
+    onDeleteHandler = (initResult: any) => {
+        initResult.controls.handleStop();
+    };
+
+    ngOnDestroy(){
```

Comment:
I believe this could be removed as `scichart-angular` should already do the same automatically